src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -932,8 +953,27 @@ public void nellaPaginaPiattaformaNotificheSiRecuperaLUltimoNumeroProtocollo() {
         }
     }
 
-    @And("Si verifica che la notifica viene creata correttamente {string}")
-    public void siVerificaCheLaNotificaVieneCreataCorrettamente(String dpFile) {
+    /**
+     * A simple object that represents the esito notifica, i.e. the return value of siVerificaEsitoNotifica.
+     */
+    class EsitoNotifica {
```

Comment:
It is not better to insert it in a specific file

Reply:
I would prefer to keep the class `EsitoNotifica` as internal to `NotificaMittentePagoPATest`, since I created the former just for internal communication between different methods inside the latter. Note the definition of `EsitoNotifica` is **inside** that of `NotificaMittentePagoPATest`.